File: evals/fixtures/dotnet/src/Billing/InvoiceService.cs
namespace Billing;

public sealed class InvoiceService
{
    public string AmountDueLabel(IEnumerable<long> amounts) => $"Amount due: {FormatMoney(amounts.Sum())}";

    private static string FormatMoney(long cents) => $"${cents / 100}.{cents % 100:00}";
}

File: evals/fixtures/laravel/app/Services/InvoiceService.php
<?php

declare(strict_types=1);

namespace App\Services;

class InvoiceService
{
    /** @param list<int> $amounts cents */
    public function amountDueLabel(array $amounts): string
    {
        return 'Amount due: ' . $this->formatMoney(array_sum($amounts));
    }

    private function formatMoney(int $cents): string
    {
        return '$' . number_format($cents / 100, 2);
    }
}
